src/test/java/ReaderWriterTest.java

```diff
@@ -1,91 +1,102 @@
-import java.util.List;
-import org.junit.Assert;
-import org.junit.Before;
+import java.util.ArrayList;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
 /* Validate that a .txt file can be properly be stored written and read from database.  */
 public final class ReaderWriterTest {
-  private static String inputFile;
-  private static String outputFile;
-  private static InputStream input;
-  private static OutputStream output;
-
-  private Database database = new Database("test");
-  private DbWriter writer = new DbWriter(database, "test", "writingIn");
-  private DbReader reader = new DbReader(database, "test", "writingIn");
-
-  @Before
-  public void setUp() {}
 
   @Test
-  public void validateDatabaseName() {
+  public void validateDatabaseName() throws IOException {
     System.out.println("validateDatabaseName()");
+    Database database = new Database("validateDatabaseName");
     /* Validate that place where data is stored is named test. */
-    System.out.println("test = " + database.getDatabase());
-    Assert.assertEquals(database.getDirectoryName(), "test");
+    System.out.println("test = " + database.getDatabaseName());
+    assertEquals("validateDatabaseName", database.getDatabaseName());
+
+    /* Delete directory.*/
   }
 
   @Test
-  public void validateWriterRunId() {
+  public void validateWriterRunId() throws IOException {
     System.out.println("validateWriterRunId()");
+    Database database = new Database("validateWriterRunId");
     /* Validate that writer's runId is test. */
-    Assert.assertEquals(writer.getRunId(), "test");
+    DbWriter writer = new DbWriter(database, "2ndTestRunId", "writingIn");
+    assertEquals("2ndTestRunId", writer.getRunId());
+
+    /* Delete directory.*/
+
   }
 
   @Test
   public void validateReaderWriterDatabaseConnect() throws IOException {
     System.out.println("validateReaderWriterDatabaseConnect()");
-    DbWriter writer2 = new DbWriter(database, "test", "writingIn");
-    DbReader reader2 = new DbReader(database, "test", "writingIn");
+
+    /*Validate that what writer wrote to database is what reader reads. */
+    /* Initialize Database, reader, and writer. */
+    Database database2 = new Database("validateReaderWriterDatabaseConnect");
+    DbWriter writer2 = new DbWriter(database2, "3rdTestRunId", "writingIn");
+    DbReader reader2 = new DbReader(database2, "3rdTestRunId", "writingIn");
 
     String writeTest = "This is a test string";
-    /*Validate that what writer wrote to dtabase is what reader reads. */
+
+    /* Write writeTest into database.*/
     try (OutputStream output1 = writer2.write()) {
       byte[] b = writeTest.getBytes();
       output1.write(b);
     }
+    /* Close outputStream. */
     writer2.finish();
 
     String readTest = null;
-    try (InputStream input = reader2.read()) {
+
+    /* Read contents of file written to database. */
+    try (Reader reader =
```

Comment:
Same here, the stream will be automatically closed and you also call `reader2.finish()`.